include/sc2api/sc2_interfaces.h

```diff
@@ -284,9 +283,8 @@ class QueryInterface {
 
     struct PlacementQuery {
         PlacementQuery() = default;
-        PlacementQuery(AbilityID ability_id, Point2D target) :
-            ability(ability_id),
-            target_pos(target) {};
+        PlacementQuery(AbilityID ability_id, Point2D target) : ability(ability_id),
+                                                               target_pos(target){};
```

Comment:
Please, leave it as is.

Reply:
Sorry, I missed that. It should be fixed. 
It was VS formatting what it thinks is Google.